jwst/stpipe/step.py

```diff
@@ -24,7 +24,8 @@
 
 SUFFIX_LIST = ['rate', 'cal', 'uncal', 'i2d', 's2d', 's3d',
                'jump', 'ramp', 'x1d', 'x2d', 'x1dints', 'calints', 'rateints']
-REMOVE_SUFFIX = '^(.+?)(_(' + '|'.join(SUFFIX_LIST) + '))?$'
+REMOVE_SUFFIX = '^(?P<root>.+?)((?P<separator>_|-)(' \
+                + '|'.join(SUFFIX_LIST) + '))?$'
```

Comment:
Are you really expecting me to verify that's the correct expression?  Aaaaaaaargh. I see an underscore and a dash separated by an or-bar buried in there, so I'm going to guess that'll do it.